### PR TITLE
23485 direct deposit widget

### PR DESCRIPTION
**Pull request by SebVortex**
## Description
1. The sentence in the note:
"Updates here will not change accounts on file for your VA health benefits." is replaced with: 'Updates here will not change accounts on file for your VA health benefits.'

2. If the applicant has bank account pre-fill information:
    a. The content This is the bank account information we have on file for you. We’ll pay your housing stipend to this account. is not displayed.
    b. The content This is the bank account information we have on file for you. We’ll send your housing payment to this account. is displayed within the "Payment Information" box.
    c. The 'Update Account' label is replaced with 'Update account information'.

3. If the applicant does not have bank account pre-fill information:
    a. The content We don't have your bank information on file. Please provide your direct deposit information below. We'll pay your housing stipend to this account. is not displayed.
    b. The content We make payments only through direct deposit, also called electronic funds transfer (EFT). Please provide your direct deposit information below. We’ll send your housing payment to this account. is displayed within the "Update bank account" box.
    c. "Account type" is required
    d. A sample check is displayed in the "Update bank account" box.
    e. The "Bank account number" is required
    f. The "Bank routing number" is required
    g. The "Bank routing number" is displayed before the "Bank account number"
    h. The "Bank routing number" label is replaced with "Bank’s 9 digit routing number".

4. The What if I don’t have a bank account? content is replaced with:
        The [Veterans Benefits Banking Program (VBBP)](https://veteransbenefitsbanking.org/) provides a list of Veteran-friendly banks and credit unions. They’ll work with you to set up an account, or help you qualify for an account, so you can use direct deposit. To get started, call one of the participating banks or credit unions listed on the VBBP website. Be sure to mention the Veterans Benefits Banking Program.

Note: The Department of the Treasury requires us to make electronic payments. If you don’t want to use direct deposit, you’ll need to call the Department of the Treasury at [888-224-2950](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/url). Ask to talk with a representative who handles waiver requests. They can answer any questions or concerns you may have.

## Original issue(s)
[department-of-veterans-affairs/va.gov-team#0000](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/23485)

## Testing done
Local testing done

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by SebVortex** [comment]
Last note from issue 23485

_After an ongoing conversation with the design team and an assessment of work vs value that could be brought to veterans with these changes, the recommendations for this issue are going to be taken as far as the schemaForm system allows. Members of the design team were reached out too but a conclusive solution addressing this specific issue was not met.

Slack thread: https://dsva.slack.com/archives/CBU0KDSB1/p1645028528355829
Support ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/37080

All attributes but 3d are to be implemented. (Specifically the placement. it's in the box but not after the account type)
(schemaForms doesn't support adding a description field between input elements)._

File: src/applications/edu-benefits/0994/components/PaymentView.jsx [hunk 1]
```diff
@@ -2,7 +2,9 @@ import _ from 'lodash';
 import React from 'react';
 import { connect } from 'react-redux';
 import { maskBankInformation, hasNewBankInformation } from '../utils';
+import { hasPrefillBankInfo } from '../pages/bankInformation';
 import AlertBox from '@department-of-veterans-affairs/component-library/AlertBox';
```
Comment (pdavies88) [inline src/applications/edu-benefits/0994/components/PaymentView.jsx:6]: Can we convert this to the `<va-alert>` web component

**Comment by pdavies88** [comment]
@andresriveratoro can I get your eyes on this since it seems like you were working with Sebastian on his fixes. 

@SebVortex is the lingering issue that you can't get the Check Image to come through?
It seems like you would need to use "view:checkImage" since it is just html you are trying to render.

I would check out:
Form 10203: `/education/other-va-education-benefits/stem-scholarship/apply-for-scholarship-form-22-10203`
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/10203/pages/directDeposit.js
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/10203/content/directDeposit.jsx
Form 1990s: `/education/other-va-education-benefits/veteran-rapid-retraining-assistance/apply-for-vrrap-form-22-1990s`
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/1990s/pages/directDeposit.jsx
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/1990s/content/directDeposit.jsx
Since direct deposit is being used elsewhere on forms in Vets-Website

Here's a convo about a similar situation from yesterday: https://dsva.slack.com/archives/CBU0KDSB1/p1645550197959899

**Comment by SebVortex** [comment]
> @SebVortex is the lingering issue that you can't get the Check Image to come through? It seems like you would need to use "view:checkImage" since it is just html you are trying to render./p1645550197959899

I'll see about that. I've been able to get it to render. The main problem was placement as specified in the story. But I'll see about that fix when I have time.

**Comment by SebVortex** [comment]
Issue being continued here: https://github.com/department-of-veterans-affairs/vets-website/compare/23485-update-direct-deposit?expand=1

New commits aren't getting picked up and tested by the automation.

**Pull request closed without merging**